packages/kit/src/runtime/client/router.js

```diff
@@ -83,7 +83,11 @@ export class Router {
 		const trigger_prefetch = (event) => {
 			const a = find_anchor(/** @type {Node} */ (event.target));
 			if (a && a.href && a.hasAttribute('sveltekit:prefetch')) {
-				this.prefetch(new URL(/** @type {string} */ (a.href)));
+        const svg = typeof a.href === 'object' && a.href.constructor.name === 'SVGAnimatedString';
+        const href = String(svg ? /** @type {SVGAElement} */ (a).href.baseVal : a.href);
+        // Include baseURI if link is inside an svg
+        const base = svg ? document.baseURI : undefined
```

Comment:
`pnpm lint` is failing because of missing semi-colon
```suggestion
        const base = svg ? document.baseURI : undefined;
```